index.ts

```diff
@@ -630,26 +768,23 @@ class ServerlessCustomDomain {
      * Logs message if SLS_DEBUG is set
      * @param message message to be printed
      */
-    public logIfDebug(message: any): void {
+    public logIfDebug(message: any, domain?: string): void {
```

Comment:
It can, totally up to the Amplify team. I can rename it but I will wait for confirmation from you.

This was an already existing method I just added the domain argument to so the user would know which domain was failing on a multi API deployment.

Reply:
I was referring to the fact that the message now always includes the word `Error` so we could make the function name show that. It's a nitpick so leaving it as is for now is fine too

Reply:
Ah, gotcha. I checked `master` it is only using for errors, but I'm inclined to keep it as is for now as it could theoretically be used for other messages. 

And there isn't a dedicated `serverless.cli` method specifically for errors:

<img width="511" alt="Screen Shot 2020-05-04 at 11 13 18 PM" src="https://user-images.githubusercontent.com/1007085/81032219-0c702400-8e5d-11ea-9505-a78f5976be30.png">

...so probably best if this is handled in its own separate commit.